package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.0.0",
   "scripts": {
     "client": "cd client && npm start",
-    "server": "nodemon server.js",
+    "server": "cd server && npm start",
     "dev": "concurrently --kill-others-on-fail \"npm run-script server\" \"npm run-script client\"",
     "start": "node server.js",
     "heroku-postbuild": "cd client && npm install && npm install --only=dev --no-shrinkwrap && npm run build"
```

Comment:
Look at this line

Reply:
Did you do `npm install` in the server directory?

Reply:
Nothing, I believe it will work the way it's suppose to be. I'll make changes when the time comes.